docs/1.getting-started/12.upgrade.md

```diff
@@ -132,6 +132,7 @@ app/
   app.vue
   router.options.ts
 layers/
+content/
```

Comment:
Let's keep this alphabetised. Would also maybe good to mention in the text (with a link to nuxt/content PR implementing it).

I'll implement unless you get to it first 🙏

Reply:
Done! I thanks 👏